src/utils/mutations.ts

```diff
@@ -303,6 +399,18 @@ export async function completeTask(classroomID: string, taskID: string, playerID
 	})
 }
 
+export async function UnsendTask(classroomID: string, taskID: string, playerID: string) {
+	// Remove `playerID` from assigned array
+
+	await updateDoc(doc(db, `classrooms/${classroomID}/tasks/${taskID}`), {
```

Comment:
I would write these two updates in a single transaction just to ensure they always happen together. I can't really think of any potential race conditions atm but this is probably just good future proofing?